app/controllers/api/v0/admin/organizations_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Api
+  module V0
+    module Admin
+      class OrganizationsController < ApiController
```

Comment:
While we haven't finalized the plans as yet, but we have already deprecated the V0 API and our goal is to remove those routes in six months or so. We've [already announced](https://developers.forem.com/api) that new features will be added on the V1 side and _not_ on the V0 side. tl;dr, you can _should_ not add new features to the V0 API.

Reply:
thanks! got that out of there